### PR TITLE
Make mock list()'s signature the same as real mock()

### PR DESCRIPTION
**Pull request by alex-hutton**
Adjusted docstring.

### CODE REVIEW

File: ixprofile_client/webservice.py [hunk 1]
```diff
@@ -149,10 +149,11 @@ def details(self, email):
...
             return response.json()
 
-    def list(self, **kwargs):
+    def list(self, *args, **kwargs):
         """
-        List all the users subscribed to the application (or ones adminable
-        by it).
+        List all the users subscribed to the application.
+
+        Kwargs are turned into a query string and sent to profile server's /user/ endpoint.
```
Comment (koterpillar) [inline ixprofile_client/webservice.py]: And args?

Comment (alex-hutton) [inline ixprofile_client/webservice.py]: no args, do you want me to remove it from the sig?

Comment (alex-hutton) [inline ixprofile_client/webservice.py]: I removed args

**Comment by koterpillar** [comment]
++

**Pull request merged**